lib/paths-provider.js

```diff
@@ -191,12 +191,17 @@ export default class PathsProvider extends EventEmitter {
 
       // Calculate distance to file
       const distanceToFile = relativePath.split(path.sep).length
+      
+      const imgRegex = new RegExp(`.(jpg|jpeg|png)$`)
```

Comment:
Why not make this list a setting? Im sure atom can support more image types, the user might also want to do the same thing?

Gifs for example come to mind.

Also is this regex case sensitive? What about for .JPEG?

Reply:
`svg` should also be on that list.

Reply:
@psykzz @OmgImAlexis 
Thank for comments.

>  make this list a setting

It's a good idea. 👍